2_process.R

```diff
@@ -50,22 +50,27 @@ p2_targets_list <- list(
   ## Data was clipped to drb before getting added to caldera.
   
   # Reach -- depth_to_bedrock data for each nhm reach buffered at 250m  
+  ## Note: In function, we transform the proj of vector to the raster (4326) to perform weighted average. Retransform to 5070 after computation at end of code chunk.  
```

Comment:
```suggestion
  ## Note: In function, we transform the proj of vector to the raster (4326) to 
  ## perform weighted average. Retransform to 5070 after computation at end of code chunk.  
```

Minor formatting suggestion so that the comment lines don't extend past the viewpoint of the screen.